### ISSUE TITLE
Violation: /usr/local/pf/logs/violation.log not present

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#3145]
It looks like the file /usr/local/pf/logs/violation.log is not created at the PacketFence installation:

````
Apr 18 11:37:47 testing-c7 pfqueue: pfqueue(16280) INFO: [mac:unknown] /usr/local/pf/logs/violation.log 2018-04-18 11:37:47: P2P Isolation (snort example) (1100006) detected on node 78:4f:43:33:45:f6 (172.20.20.169) (pf::action::action_log)
Apr 18 11:37:47 testing-c7 pfqueue: pfqueue(16280) FATAL: [mac:unknown] Unable to open /usr/local/pf/logs/violation.log for append: Permission denied at /usr/local/pf/lib/pf/violation.pm line 448.
 (pf::action::action_log)
Apr 18 11:37:47 testing-c7 pfqueue: pfqueue(16280) ERROR: [mac:unknown] Unable to open /usr/local/pf/logs/violation.log for append: Permission denied at /usr/local/pf/lib/pf/violation.pm line 448.
 (pf::api::can_fork::notify)
Apr 18 11:37:48 testing-c7 pfqueue: pfqueue(21887) INFO: [mac:78:4f:43:33:45:f6] [78:4f:43:33:45:f6] DesAssociating mac on switch (10.0.0.14) (pf::api::desAssociate)
Apr 18 11:37:48 testing-c7 pfqueue: pfqueue(21887) INFO: [mac:78:4f:43:33:45:f6] deauthenticating (pf::Switch::radiusDisconnect)
````

**Comment by MiCRoPhoBIC** [comment]
This issue still exists in PacketFence-8.1

**Comment by nqb** [comment]
File is present on a fresh install (CentOS 7) with devel packages (8.1.9).

**Comment by MiCRoPhoBIC** [comment]
Maybe it was fixed after 8.1 release? On fresh PF 8.1 package install on Debian 8 the file is missing.

**Comment by nqb** [comment]
@MiCRoPhoBIC: I tested on CentOS7. Agree with you, file is missing.
